### PR TITLE
happy new year 2021 GLPI :tada:

### PR DESCRIPTION
**Pull request by flegastelois**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A

### CODE REVIEW

**Review by orthagh — APPROVED** [review]

File: inc/impact.class.php [hunk 1]
```diff
@@ -5,7 +5,34 @@
 }
 
 /**
- * @since 9.5.0
+ * ---------------------------------------------------------------------
```
Comment (cedric-anne) [inline inc/impact.class.php:8]: This line should not be removed, header should be on line 2.

File: inc/impactcompound.class.php [hunk 2]
```diff
@@ -5,7 +5,34 @@
 }
 
 /**
- * @since 9.5.0
+ * ---------------------------------------------------------------------
```
Comment (cedric-anne) [inline inc/impactcompound.class.php:8]: Same remark

File: inc/impactcontext.class.php [hunk 3]
```diff
@@ -5,7 +5,34 @@
 }
 
 /**
- * @since 9.5.0.
+ * ---------------------------------------------------------------------
```
Comment (cedric-anne) [inline inc/impactcontext.class.php:8]: Same remark

File: inc/impactitem.class.php [hunk 4]
```diff
@@ -5,7 +5,34 @@
 }
 
 /**
- * @since 9.5.0.
+ * ---------------------------------------------------------------------
```
Comment (cedric-anne) [inline inc/impactitem.class.php:8]: Same remark

File: inc/impactrelation.class.php [hunk 5]
```diff
@@ -5,7 +5,34 @@
 }
 
 /**
- * @since 9.5.0
+ * ---------------------------------------------------------------------
```
Comment (cedric-anne) [inline inc/impactrelation.class.php:8]: Same remark

File: inc/vobject.class.php [hunk 6]
```diff
@@ -5,7 +5,34 @@
 }
 
 /**
- * @since 9.5.0
+ * ---------------------------------------------------------------------
```
Comment (cedric-anne) [inline inc/vobject.class.php:8]: Same remark

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**